liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/StartDebugMojoSupport.java

```diff
@@ -72,6 +72,8 @@
     private static final String LIBERTY_CONFIG_MAVEN_PROPS = "(^liberty\\.(env|jvm|bootstrap|var|defaultVar)\\.).+";
     private static final Pattern pattern = Pattern.compile(LIBERTY_CONFIG_MAVEN_PROPS);
 
+    private static boolean configFilesCopied = false;
```

Comment:
I just want to make sure that this is intentionally a `static` variable? Does it need to be? I'm not sure how it works between one goal execution and the next with maven.